addons/stock_account/models/stock_lot.py

```diff
@@ -0,0 +1,136 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from ast import literal_eval
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools import float_compare, float_round
+
+
+class StockLot(models.Model):
+    _inherit = 'stock.lot'
+
+    value_svl = fields.Float(compute='_compute_value_svl', compute_sudo=True)
+    quantity_svl = fields.Float(compute='_compute_value_svl', compute_sudo=True)
+    avg_cost = fields.Monetary(string="Average Cost", compute='_compute_value_svl', compute_sudo=True, currency_field='company_currency_id')
+    total_value = fields.Monetary(string="Total Value", compute='_compute_value_svl', compute_sudo=True, currency_field='company_currency_id')
+    company_currency_id = fields.Many2one('res.currency', 'Valuation Currency', compute='_compute_value_svl', compute_sudo=True)
+    stock_valuation_layer_ids = fields.One2many('stock.valuation.layer', 'lot_id')
+    standard_price = fields.Float(
```

Comment:
should be a compute with inverse function no? In order to not store nor synchronize it

Reply:
Nope since we could use for inventory adjustement with import the lot and their value before doing the inventory